### ISSUE TITLE
Review of WetKit Images - Adding a Caption & Make Accessible

### ISSUE THREAD

**Issue by mgifford** [wet-boew/wet-boew-drupal#411]
I'm wanting to add a caption facility similar to  http://drupal.org/project/jcaption to WET's image handling. It's great that there is the custom WETKit Lightbox Display that can be configured. However, the image alone isn't sufficient in this context.

Actually, there is an image here:
http://wet-boew.github.com/wet-boew/demos/lightbox/lightbox-eng.html

On a side note, I'm not sure how to close the Lightbox without a mouse....

Also in the example from WET it is easy to move between images in the gallery.  Not sure that's possible with a few images that are attached to a node at the moment.

**Comment by pjackson28** [comment]
@mgifford The escape key is one way you can close the lightbox with keyboard.

**Comment by mgifford** [comment]
@pjackson28 With the gallery in the example above I was able to tab to both the prev/next slide as well as close the box with just the tab key.

**Comment by pjackson28** [comment]
@mgifford You can use tabs in WET lightbox as well. Just tab to the close button and hit enter or space bar. Can use tab to move forward or back in the same way. Necessary to do this as the lightbox supports the display of regular content as well so need to be able to tab through any links or form elements that are included.

**Comment by sylus** [comment]
Hey @mgifford if you specify the title attribute text does appear as a caption in the lightbox. Are you referring to a caption in non lightbox mode?

**Comment by mgifford** [comment]
If you add an image field to an existing content type you don't have the option of specifying a title.  If you have a unique image content type then this is possible (although I haven't tried it).

**Comment by sylus** [comment]
Fixed please reopen if anything missing :)

**Comment by sylus** [comment]
Noticing a missing alt tag and array to string conversion, next commit will fix this.
